src/RandNumGenerator.java

```diff
@@ -17,20 +17,33 @@ public static void main (String[] args) {
         System.out.println("The lower bound is: " + lowerNum);
 
         //getting upper bound of range
-        System.out.println("Please input the upper bound of your range (i.e. 10)");
-        int upperNum = mrScanner.nextInt();
-        System.out.println("The upper bound is: " + lowerNum);
-
-        //getting amount of numbers
-        System.out.println("How many random numbers would you like?");
-        int amtNum = mrScanner.nextInt();
-        System.out.println("Generating " + amtNum + " number(s):");
-
-        int i = amtNum;
-        while (i > 0) {
-            int randNum = lowerNum + rand.nextInt(upperNum - lowerNum);
-            System.out.println(randNum);
-            i--;
+        int invalNum = 1;
+        int upperNum;
+        while (invalNum != 0) {
+            System.out.println("Please input the upper bound of your range (i.e. 10)");
+            upperNum = mrScanner.nextInt();
+            if (upperNum < lowerNum) {
```

Comment:
What happen when `upperNum equals lowerNum`

Reply:
I didn't even think about that!